### ISSUE TITLE
Imported product with picture( wp-all import) using the plugin multiside-global-media  

### ISSUE THREAD

**Issue by betimademi** [bueltge/multisite-global-media#82]
I have an issue when i try to import a product with multisite-global-media activated, the product is imported but the picture is imported as product gallery and is not showing in the Woocoomerce-> All products 

![Screen Shot 2019-04-03 at 02 56 27](https://user-images.githubusercontent.com/23141707/55445725-fd190a00-55bc-11e9-87ea-912337deb610.png)

![Screen Shot 2019-04-03 at 02 56 54](https://user-images.githubusercontent.com/23141707/55445726-fd190a00-55bc-11e9-8c5e-da2fe6c1e6fa.png)

**Comment by bueltge** [comment]
If you import this pictures, in which media gallery is it implemented. I mean the import tool should import in the site, there you current use, not inside the Global Media Site, there is defined.

**Comment by betimademi** [comment]
In site ID 1 when i do the import with picture  it is imported in Media Library but also when i check in site ID 2 in the Global media the picture is there also, so i assume the import in site ID 1 imports the picture in Media Library and also appears in  Global Media.

**Comment by bueltge** [comment]
The Global Media is a store in one! site of the network, on default Site ID 1. The plugin share only the images about all sites from site 1. So if you import it in Site ID 1 and the Global Media Plugin is active, the media files should visible in all sites in the tab Global Media.

**Comment by betimademi** [comment]
Yes it is true but my point is that i have 3 multi language sites in wordpress and by default site with ID 1 shares images across other network using GLOBAL MEDIA, I want to import products with images in site 1, because 2 other multisites share same products and images but with different language i want to import products without image on two other sites and to point to that GLOBAL MEDIA FOLDER to fetch the images imported from site 1. (The import plugin im using is WP ALL IMPORT). So my issue was that the import of products with image is imported in site 1 and it is shown in GLOBAL MEDIA but the image it doesnt show when i go to Products -> Show all products, it is mentioned below with screenshot. When i go inside the product on the right side the image it is fetched as Product gallery not as Product Image ! I think that is the problem that the image its not showing ! Maybe it has to do with thumbnails  @bueltge
